client/src/components/QualityManagement/Parts.jsx

```diff
@@ -0,0 +1,328 @@
+            mr={3}
+          >
+            Add
+          </Button>
+          <Button onClick={onClose}>Cancel</Button>
+        </ModalFooter>
+      </ModalContent>
+    </Modal>
+  );
+
+  if (isSuccess && data.data.length === 0) {
+    return (
+      <>
+        <Heading size="xl" textAlign="center" mt={5}>
+          No Defects
+        </Heading>
+        {addDefectForm()}
+        <Link onClick={onOpen}>
+          <NoResultImage />
+        </Link>
+      </>
+    );
+  }
+
+  return (
+    <Box overflowX="auto">
+      <IconButton
+        colorScheme="blue"
+        variant="outline"
+        aria-label="add"
+        float="right"
+        m={2}
+        icon={<SmallAddIcon />}
+        onClick={onOpen}
+      />
+      <IconButton
+        colorScheme="blue"
+        variant="outline"
+        aria-label="delete"
+        float="right"
+        m={2}
+        icon={<DeleteIcon />}
+        onClick={handleDelete}
+        isDisabled={selected.length === 0}
+      />
+      <TableContainer component={Paper}>
+        <Table minWidth="unset" width="100%" variant="striped" colorScheme="light">
+          <Thead>
+            <Tr>
+              <StyledTableHeader>
+                <Checkbox
+                  isIndeterminate={selected.length > 0 && selected.length < data.data.length}
+                  isChecked={data.data.length > 0 && selected.length === data.data.length}
+                  onChange={handleSelectAllClick}
+                >
+                  Ticket ID
+                </Checkbox>
+              </StyledTableHeader>
+              <StyledTableHeader>Product Name</StyledTableHeader>
+              <StyledTableHeader>Defect Type</StyledTableHeader>
+              <StyledTableHeader>Status of Defect</StyledTableHeader>
+              <StyledTableHeader>Desc of Solution</StyledTableHeader>
+            </Tr>
+          </Thead>
+          <Tbody>
+            {isSuccess &&
+              data.data
+                .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+                .map((defect) => (
+                  <Fragment key={defect._id}>
+                    <StyledTableRow
+                    //selected={isSelected(defect._id)}
+                    // onClick={(event) => handleClick(event, defect._id)}
+                    >
+                      <StyledTableCell>
+                        <Checkbox
+                          onChange={(event) => handleOnChange(event, defect._id)}
+                          isChecked={isSelected(defect._id)}
+                        >
+                          {defect.id}
+                        </Checkbox>
+                      </StyledTableCell>
+                      <StyledTableCell>{defect.partName}</StyledTableCell>
+                      <StyledTableCell>
+                        <Tag
+                          size="md"
+                          variant="solid"
+                          colorScheme={defect.type == 'Broken' ? 'red' : 'orange'}
```

Comment:
```suggestion
                          colorScheme={defect.type === 'Broken' ? 'red' : 'orange'}
```

To prevent JavaScript coercing